torch/_dynamo/variables/builtin.py

```diff
@@ -1172,6 +1172,25 @@ def call_id(self, tx, *args):
         else:
             unimplemented(f"call_id with args {args}")
 
+    def _raw_value_to_format(self, value):
```

Comment:
I think another way for this is just implement ```__str__``` for every variable, most of the case you can just call ```.as_python_constant()```

Reply:
Well yeah, except when they are containers and tensors. I'd expect that we'll also support dicts, e.g., and that's easy to add. There seem to be many other variable kinds, though. Do you suggest I support all of them in one go here?

Reply:
I think you can add ```__str__``` to base class ```VariableTracer```, and return ```as_python_constant``` if it is python constant. It can cover most of the cases, for the other variables, just add the separately.